src/index.spec.ts

```diff
@@ -1342,4 +1342,25 @@ describe('EppoClient config', () => {
     expect(retryManager['config']['maxRetryDelayMs']).toEqual(3);
     expect(retryManager['config']['maxRetries']).toEqual(4);
   });
+
+  it('handles empty precomputed configuration string', () => {
+    // Test with throwOnFailedInitialization = true (default)
+    expect(() =>
+      offlinePrecomputedInit({
+        precomputedConfiguration: '',
```

Comment:
is the exception specifically an empty string or any malformed configuration json?

Reply:
The exception specifically in mind was an empty string which seems likely to be a fallback in loading the configuration string, i.e.

```
let configuration = '';
... and then set configuration to precomputed string
```

but this would also catch any other string that cannot be parsed to json, which is useful too. The precomputed configuration should always be valid json